cli/server/spreadsheetRoutes.ts

```diff
@@ -877,55 +962,172 @@ function exportAsCSV(controls: any[], metadata: any, res: express.Response) {
-					? 'Mappings Count'
-					: key === 'mappings'
-						? 'Mappings'
-						: key.replace(/-/g, ' ').replace(/\b\w/g, (l) => l.toUpperCase()));
-			const value = control[key];
-
-			// Special handling for mappings
-			if (key === 'mappings' && Array.isArray(value)) {
-				exportControl[displayName] = value
-					.map(
-						(m: any) =>
-							`${m.status}: ${m.description.substring(0, 100)}${m.description.length > 100 ? '...' : ''}`
-					)
+	// Handle family field second if it exists
+	if (allFields.has('family')) {
+		const familySchema = fieldSchema['family'];
+		const displayName = familySchema?.original_name || 'Family';
+		fieldMapping.push({ fieldName: 'family', displayName });
+		usedDisplayNames.add(displayName.toLowerCase());
+		allFields.delete('family');
+	}
+
+	// Handle remaining fields, checking if any should be replaced by mappings
+	Array.from(allFields)
+		.filter((field) => field !== 'mappings' && field !== 'mappings_count')
+		.sort()
+		.forEach((field) => {
+			const schema = fieldSchema[field];
+			const defaultDisplayName =
+				schema?.original_name || field.replace(/-/g, ' ').replace(/\b\w/g, (l) => l.toUpperCase());
+
+			// Check if this field should show mappings data instead
+			let isMappingColumn = false;
+			let finalDisplayName = defaultDisplayName;
+
+			// Check if mappings should be exported to this field's column
+			if (columnMappings['mappings']) {
+				const targetDisplayName = columnMappings['mappings'];
+				// Check if this field's display name matches the target
+				if (defaultDisplayName.toLowerCase() === targetDisplayName.toLowerCase()) {
+					isMappingColumn = true;
+					finalDisplayName = targetDisplayName; // Use the exact target name
+				}
+			}
+
+			if (!usedDisplayNames.has(finalDisplayName.toLowerCase())) {
+				fieldMapping.push({
+					fieldName: field,
+					displayName: finalDisplayName,
+					isMappingColumn
+				});
+				usedDisplayNames.add(finalDisplayName.toLowerCase());
+			}
+		});
+
+	// Add regular mappings column if not being exported to another column
+	if (
+		allFields.has('mappings') &&
+		(!columnMappings['mappings'] || columnMappings['mappings'] === 'Mappings')
+	) {
+		fieldMapping.push({ fieldName: 'mappings', displayName: 'Mappings' });
+	}
+
+	// Prepare data for Excel with the field mapping
+	const worksheetData = controls.map((control) => {
+		const exportControl: any = {};
+
+		fieldMapping.forEach(({ fieldName, displayName, isMappingColumn }) => {
+			let value;
+
+			if (isMappingColumn) {
+				// This column should show mappings data, fallback to original field if no mappings
+				const mappingsValue = control['mappings'];
+				if (Array.isArray(mappingsValue) && mappingsValue.length > 0) {
+					// Show mappings justification - collect all non-empty justifications
+					const mappingsStr = mappingsValue
+						.map((m: any) => m.description || m.justification || '')
+						.filter((desc: string) => desc && desc.trim() !== '')
+						.join('\n');
```

Comment:
I think for now this makes sense and I prefer this, but just to confirm - this doesn't include the status anymore. I think the status is more for a dev tracking UUID inclusion (in my opinion) so excluding here makes sense to me.

Reply:
okay makes sense